### PR TITLE
Carrier rebalance

### PR DESCRIPTION
**Pull request by Naaanii**
## About The Pull Request

Halves the initial hugger stun duration and gives Carrier a slight slash damage increase instead.

## Why It's Good For The Game

Hugger stun too strong. Halves the initial stun duration and makes Carrier need to act in a shorter time frame. Also, lets marines react much quicker after the hug.

## Changelog
:cl: Naaanii
balance: Hugger unconscious duration halved. Slash damage buffed.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/mob/living/carbon/xenomorph/castes/carrier/castedatum_carrier.dm [hunk 1]
```diff
@@ -143,7 +143,7 @@
 	ancient_message = "We are the master of huggers. We shall throw them like baseballs at the marines!"
 
 	// *** Melee Attacks *** //
-	melee_damage = 20
+	melee_damage = 24
```
Comment (Pariah919) [inline code/modules/mob/living/carbon/xenomorph/castes/carrier/castedatum_carrier.dm:146]:
```suggestion
	melee_damage = 22
```
slash damage is in 2 stages, this is no exception

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm [hunk 2]
```diff
@@ -189,7 +189,6 @@
 
 	owner.visible_message("<span class='danger'>\The [owner] strikes [target] with [flavour] precision!</span>", \
 	"<span class='danger'>We strike [target] with [flavour] precision!</span>")
-	target.ParalyzeNoChain(1 SECONDS)
 	target.adjust_stagger(staggerslow_stacks)
 	target.add_slowdown(staggerslow_stacks)
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:193]: nic code you banana

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm [hunk 3]
```diff
@@ -211,7 +210,6 @@
 
 	owner.visible_message("<span class='danger'>\The [owner] strikes [target] with [flavour] precision!</span>", \
 	"<span class='danger'>We strike [target] with [flavour] precision!</span>")
-	target.ParalyzeNoChain(1 SECONDS)
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:214]: what a big noob

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm (hunk 2 above)
Comment (Naaanii) [reply code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:193]: Oops

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm (hunk 3 above)
Comment (Naaanii) [reply code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:214]: SHUSH

**Comment by Naaanii** [comment]
Ancient damage lowered to 22 with Pariah's review.

**Comment by RuneQuester** [comment]
and now carrier is even more useless

**Comment by qfmysteryman23** [comment]
Huggers barely even work any more, so why the fuck should we worry about hugger stun timer when people barely ever get hugged in the first place?

**Comment by Naaanii** [comment]
> Huggers barely even work any more, so why the fuck should we worry about hugger stun timer when people barely ever get hugged in the first place?


https://user-images.githubusercontent.com/73274515/107042951-56b97380-67e8-11eb-8528-fab51440ce5b.mp4

**Comment by Naaanii** [comment]
Will reopen, if the Neuro rework does not go through.

**Comment by brasscastle187** [comment]
> 
> 
> > Huggers barely even work any more, so why the fuck should we worry about hugger stun timer when people barely ever get hugged in the first place?
> 
> Free.hugs.mp4

Lone marine running 15 tiles away from only visible teammate being used to justify balance explains a lot.

**Pull request closed without merging**